tests/Integration/ScriptRuntime/ProcessExecutorTest.php

```diff
@@ -252,7 +252,7 @@ public function test_deferred_commands_get_executed_even_with_error_in_between()
         } catch (ExecutionErrorException $e) {
         }
         $executionTime = microtime(true) - $beginExecution;
-        self::assertInstanceOf(ExecutionErrorException::class, $e);
+        $this->assertInstanceOf(ExecutionErrorException::class, $e);
```

Comment:
Previous implementation was correct.
```suggestion
        self::assertInstanceOf(ExecutionErrorException::class, $e);
```

Reply:
I'm not sure what you are aiming for. This project has previously used the method invocation off assertions, not the static ones.

Reply:
`assertInstanceOf` is a static method by definition and should be called statically therefore.

Reply:
The guy writing phpunit fails this assertion :) https://phpunit.readthedocs.io/en/9.3/writing-tests-for-phpunit.html

The reason why this is the case in the B2B-Suite is that earlier versions of PHPStorm did not support `$this`. 

But since php-cs-fixer can do this, it's already changed.